independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/buildtool/maven/codestart.yml

```diff
@@ -4,6 +4,7 @@ fallback: true
 language:
   base:
     shared-data:
+      generate-native: true
```

Comment:
What does this do?

Reply:
I put an IF block around pom stuff that deals with native build.  Azure does not support native so I don't want it added to the pom.xml.

Reply:
Ah nice!